core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -509,6 +509,7 @@ public static long getUpdateInterval() {
 
     public static int[] getUpdateTimeOfDay() {
         String datetime = prefs.getString(PREF_UPDATE_INTERVAL, "");
+
```

Comment:
Please remove the change. The fewer unrelated files this touches, the cleaner the git history ;)